tests/conftest.py

```diff
@@ -129,7 +129,14 @@ def make_local_paths(
 
     for ds, path in zip(datasets, full_paths):
         save_method = getattr(ds, method)
-        save_method(path, **kwargs)
+        # Question: Should we enable explicity chunking when saving to netcdf?
+        # by default it is encoding data with chunks, such as "chunks": [2,18,36].
+        # It might be useful to set chunking explicitly.
+        # encoding = {
+        #     'foo': {'chunksizes': (1, 2, 2)},
+        #     'bar': {'chunksizes': (1, 2, 2)}
+        # }
```

Comment:
⬆️

Reply:
I don't have strong opinion either way. I guess if it helps this immediate test case then yes, otherwise maybe just leave it as is.

Reply:
I started to right up an issue about this, to create a test that explicitly tests for chunks in the origin netcdf being replicated in destination kerchunk, but I'm not sure if that's a test more appropriate for kerchunk or pangeo-forge-recipes 🤔

Reply:
Sounds like a kerchunk test, as you suggest. Might be useful for us to figure out the next steps around merge-func disambiguation over there, as things might make more sense at that point https://github.com/fsspec/kerchunk/issues/416#issuecomment-1937817235